mriqc/interfaces/bids.py

```diff
@@ -85,6 +90,11 @@ def __setattr__(self, key, value):
                 self._outputs[key] = value
             super(IQMFileSinkInputSpec, self).__setattr__(key, value)
 
+        if key=='subject_id':
```

Comment:
This should not be necessary: `subject_id` is defined as a mandatory input, so you'll need to workaround it by setting a mocked subject id from outside.